src/app/medInria/medMainWindow.cpp

```diff
@@ -348,30 +343,34 @@ void medMainWindow::menuCapture(QMenuBar * menu_bar)
 
 }
 
+/**
+ * @brief The Settings menu holds application settings such as start-up or data sources settings
+ * 
+ * @param menu_bar 
+ */
 void medMainWindow::menuSettings(QMenuBar * menu_bar)
 {
-    // --- Settings menu
     QMenu *menuSettings = menu_bar->addMenu("Settings");
 
-    menuSettings->addAction(tr("General"));
-    menuSettings->addSeparator();
     QAction *actionDataSources = menuSettings->addAction(tr("Data Sources"));
-
     connect(actionDataSources, &QAction::triggered, this, &medMainWindow::onShowDataSources);
+
+    QAction *actionAreaSettings = menuSettings->addAction(tr("&Startup"));
+    connect(actionAreaSettings, &QAction::triggered, this, &medMainWindow::onShowAreaSettings);
 }
 
 void medMainWindow::menuAbout(QMenuBar * menu_bar)
 {
 
     // --- About menu
-    QMenu *menuAbout = menu_bar->addMenu("?");
+    QMenu *menuAbout = menu_bar->addMenu("Help");
 
     QAction *actionAbout = menuAbout->addAction(tr("About the application"));
     QAction *actionAuthors = menuAbout->addAction(tr("Au&thors"));
     QAction *actionReleaseNotes = menuAbout->addAction(tr("&Release Notes"));
     QAction *actionLicense = menuAbout->addAction(tr("&License"));
     menuAbout->addSeparator();
-    QAction *actionHelp = menuAbout->addAction(tr("&Help"));
+    QAction *actionHelp = menuAbout->addAction(tr("&Documentation"));
```

Comment:
I don't like Documentation. Online Documentation or online help.

Reply:
Done, i updated the PR

Reply:
Thanks